scripts/annotations_import.py

```diff
@@ -0,0 +1,174 @@
+import csv
+from docopt import docopt
+import logging
+import os
+import re
+from tqdm import tqdm
+from web_monitoring import db
+from web_monitoring.db import Client
+
+logger = logging.getLogger(__name__)
+log_level = os.getenv('LOG_LEVEL', 'WARNING')
+logger.setLevel(logging.__dict__[log_level])
+
+class DictReaderStrip(csv.DictReader):
+    @property
+    def fieldnames(self):
+        return [name.strip() for name in super().fieldnames]
+
+def read_csv(csv_path):
+    with open(csv_path, newline='') as csvfile:
+        reader = DictReaderStrip(csvfile)
+        for row in reader:
+            yield row
+
+DIFF_URL_REGEX = re.compile('^.*\/page\/(.*)/(.*)\.\.(.*)')
```

Comment:
Missed this earlier — there’s some unnecessary/malformed escaping here. It works in practice, but could be a little cleaner. Using a raw string might be a good idea, too:

```suggestion
DIFF_URL_REGEX = re.compile(r'^.*/page/(.*)/(.*)\.\.(.*)')
```

Reply:
Done, thanks for the catch.